src/nav/infrastructure.yml

```diff
@@ -515,28 +515,22 @@ pages:
             path: /docs/infrastructure/infrastructure-integrations/cloud-integrations/metric-data-gaps-cloud-integrations
   - title: Elastic Container Service integration
     pages:
-      - title: Get started
-        pages:
-          - title: Intro to ECS integration
-            path: /docs/infrastructure/elastic-container-service-integration/get-started/introduction-amazon-ecs-integration
-      - title: Installation
-        pages:
-          - title: Install ECS integration
-            path: /docs/infrastructure/elastic-container-service-integration/installation/install-ecs-integration
-          - title: Uninstall ECS integration
-            path: /docs/infrastructure/elastic-container-service-integration/installation/uninstall-ecs-integration
-      - title: Understand use data
-        pages:
-          - title: Understand and use ECS data
-            path: /docs/infrastructure/elastic-container-service-integration/understand-use-data/understand-use-ecs-data
-          - title: Recommended ECS alerts
-            path: /docs/infrastructure/elastic-container-service-integration/understand-use-data/ecs-integration-recommended-alert-conditions
-      - title: Troubleshooting
-        pages:
-          - title: Verbose logs
-            path: /docs/infrastructure/elastic-container-service-integration/troubleshooting/ecs-integration-troubleshooting-generate-verbose-logs
-          - title: No data appears
-            path: /docs/infrastructure/elastic-container-service-integration/troubleshooting/ecs-integration-troubleshooting-no-data-appears
+    - title: Intro to the ECS integration
+      path: /docs/infrastructure/elastic-container-service-integration/introduction-amazon-ecs-integration
+    - title: Install the ECS integration
+      path: /docs/infrastructure/elastic-container-service-integration/install-ecs-integration
+    - title: Understand and use ECS data
+      path: /docs/infrastructure/elastic-container-service-integration/understand-use-ecs-data
+    - title: Recommended ECS alerts
+      path: /docs/infrastructure/elastic-container-service-integration/ecs-integration-recommended-alert-conditions
+    - title: Troubleshooting
+      pages:
+        - title: No data appears
+          path: /docs/infrastructure/elastic-container-service-integration/troubleshooting/ecs-integration-troubleshooting-no-data-appears
+        - title: Generate verbose logs
+          path: /docs/infrastructure/elastic-container-service-integration/troubleshooting/ecs-integration-troubleshooting-generate-verbose-logs
+    - title: Uninstall the ECS integration
```

Comment:
Everything looks fine, but my gut is that troubleshooting so be the least item in this category.